### PR TITLE
Create search box form

### PR DESCRIPTION
**Pull request by CBinh710**
https://github.com/users/CBinh710/projects/2/views/1?pane=issue&itemId=72090758
image

### CODE REVIEW

File: practice-one/src/components/introduction.html [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
+      <span class="accent-color">Delivery</span>.
+    </h1>
+    <img class="header-decor" src="./assets/images/decor.png" alt="decor flash">
+    <p class="para-introduction">
       Best cooks and best delivery guys all at your service. Hot tasty food will reach you in 60 minutes.
     </p>
+    <div class="search-food-container">
+      <img src="./assets/images/search-icon-black.svg">
+      <input type="text" placeholder="Search food">
+      <img src="./assets/images/search-icon-white.svg">
```
Comment (hungdo-agilityio) [inline practice-one/src/components/introduction.html]: for empty tag should have `/` at the end

File: practice-one/src/styles/components/introduction.css [hunk 2]
```diff
@@ -52,6 +52,49 @@
   font-weight: 400;
 }
 
+
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/introduction.css]: remove redundant empty line, recheck for all

File: practice-one/src/styles/components/introduction.css [hunk 3]
```diff
@@ -52,6 +52,49 @@
...
+.search-food-container {
+  display: flex;
+  align-items: center;
+  border: solid 0.5px;
+  border-color: var(--color-dark);
+  border-radius: 100px;
+  width:471px;
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/introduction.css]: Recheck the format

**Review by hungdo-agilityio — APPROVED** [review]

File: practice-one/src/components/introduction.html (hunk 1 above)
Comment (CBinh710) [reply practice-one/src/components/introduction.html]: i understood , thanks

File: practice-one/src/styles/components/introduction.css (hunk 2 above)
Comment (CBinh710) [reply practice-one/src/styles/components/introduction.css]: i removed it thanks

File: practice-one/src/styles/components/introduction.css (hunk 3 above)
Comment (CBinh710) [reply practice-one/src/styles/components/introduction.css]: Thanks for your comment

**Pull request merged**